apps/web/src/components/TrendingItemSuggestion/TrendingItemSuggestion.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
     trendingSuggestionHeading: string;
     trendingSuggestionSummary: string;
     targetSubreddit: {
-      subRedditIcon: React.ReactElement;
+      subRedditIcon: string;
```

Comment:
Just a reminder for us all to make a ticket to figure out how to unstringify the icons after being fetched with axios. Then we could add these icons back in.